web/Services/MailService.cs

```diff
@@ -31,26 +31,27 @@ public MailService(IOptions<EmailConfigOptions> config)
         string msgToName = ""
```

Comment:
We should probably make message body subject required arguments, and put a sensible value in for the subject. Can be another PR though

Reply:
Can I add it onto another ticket that I'm working on? (the one that's dealing with sending email exceptions)

Reply:
Sure